### ISSUE TITLE
 Navbar Login Button Hidden by Light/Dark Mode Button

### ISSUE THREAD

**Issue by lovelymahor** [Suchitra-Sahoo/AgriLearnNetwork#1274]
### Describe the feature

The login button in the navbar is being hidden by the light/dark mode toggle button. This affects user navigation and accessibility, as users are unable to easily find and click the login button.

### Add ScreenShots

![Screenshot (276)](https://github.com/user-attachments/assets/b2bc5250-6aeb-4b36-b29c-41940a3fd7bc)

@su Please assign it to me I want to work on this issue. Also, please add the label GSSOC and the level that you find appropriate for this issue. thank you!

### Record

- [X] I agree to follow this project's Code of Conduct
- [X] I'm a GSSOC'24 contributor
- [X] I want to work on this issue
